lib/publiccloud/basetest.pm

```diff
@@ -33,22 +33,22 @@ sub provider_factory {
 
         if ($args{service} eq 'ECR') {
             $provider = publiccloud::ecr->new(
-                key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
-                key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
+                # key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
+                # key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
```

Comment:
Just for the record this should be removed before being merged.

Reply:
Thanks, yes cleaning up this mess

Reply:
Also region and username can be moved too to the each provider

Reply:
fixed